diff --git a/httpd-default.conf b/vulnerable-httpd.conf
index c9e61c7..111ad62 100644
--- a/httpd-default.conf
+++ b/vulnerable-httpd.conf
@@ -184,7 +184,7 @@ LoadModule autoindex_module modules/mod_autoindex.so
        #LoadModule cgid_module modules/mod_cgid.so
 </IfModule>
 <IfModule mpm_prefork_module>
-       #LoadModule cgi_module modules/mod_cgi.so
+       LoadModule cgi_module modules/mod_cgi.so
 </IfModule>
 #LoadModule dav_fs_module modules/mod_dav_fs.so
 #LoadModule dav_lock_module modules/mod_dav_lock.so
@@ -247,7 +247,7 @@ ServerAdmin you@example.com
 #
 <Directory />
     AllowOverride none
-    Require all denied
+    #Require all denied
 </Directory>

 #